multimodal-dataset/intro_vertex_multimodal_dataset.ipynb

```diff
@@ -0,0 +1,933 @@
+        ") -> datasets.GeminiTemplateConfig:\n",
+        "    \"\"\"Helper method to create a GeminiTemplateConfig object for single-turn cases.\n",
+        "    Args:\n",
+        "\n",
+        "    prompt: User input. Required.\n",
+        "    response: Model response to user input. Optional.\n",
+        "    system_instructions: System instructions for the model. Optional.\n",
+        "    field_mapping: Mapping of placeholders to dataset columns. Optional.\n",
+        "    \"\"\"\n",
+        "    gemini_example = construct_gemini_example(\n",
+        "        prompt=prompt, response=response, system_instructions=system_instructions\n",
+        "    )\n",
+        "    return datasets.GeminiTemplateConfig(\n",
+        "        gemini_example=gemini_example, field_mapping=field_mapping\n",
+        "    )"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "aKw7PCHpXwgn"
+      },
+      "source": [
+        "## User Journey Demo\n",
+        "\n",
+        "The user journey demonstrated here contains the following steps:\n",
+        "\n",
+        "1. Create Dataset\n",
+        "2. Assemble the dataset with a template and inspect assembly\n",
+        "3. Run a validation for tuning\n",
+        "4. Estimate Resources for tuning\n",
+        "5. Run tuning\n",
+        "6. Run batch prediction"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "HybHbfY85MnR"
+      },
+      "source": [
+        "### 1. Create a dataset from a Pandas or BigFrames DataFrame"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "fqewmHHY7TYO"
+      },
+      "source": [
+        "We prepared a DataFrame `training_set` with two columns:\n",
+        "\n",
+        "*   `image_uris`: GCS URIs of flower images\n",
+        "*   `labels`: Flower label (five flower categories, one label per image)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "8DxI6amF8ok1"
+      },
+      "outputs": [],
+      "source": [
+        "flower_uri = training_set[\"image_uris\"].iloc[0]\n",
+        "flower_label = training_set[\"labels\"].iloc[0]\n",
+        "\n",
+        "display(get_gcs_image(flower_uri))\n",
+        "print(f\"Image URI: {flower_uri}\")\n",
+        "print(f\"Flower label: {flower_label}\")\n",
+        "training_set.head()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "jSxMKdBiId8r"
+      },
+      "source": [
+        "Let's create a Vertex multimodal dataset from the prepared DataFrame."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Xc37pkL1SnxL"
+      },
+      "outputs": [],
+      "source": [
+        "flowers = datasets.MultimodalDataset.from_pandas(dataframe=training_set)\n",
```

Comment:
I kept getting an error saying that `from_pandas()` needs a parameter for bigquery dataset. What should be used here?

Reply:
Tested in Colab, everything works